### ISSUE TITLE
Deprecated qiskit.extensions.unitary 

### ISSUE THREAD

**Issue by evmckinney9** [CQCL/pytket-qiskit#184]
According to this PR (https://github.com/Qiskit/qiskit/pull/10725), will receive the error `ModuleNotFoundError: No module named 'qiskit.extensions.unitary'`

```python
from qiskit.extensions.unitary import UnitaryGate
```

should be replaced with 
```python
from qiskit.circuit.library import UnitaryGate
```
for Qiskit version 0.45.0

**Comment by CalMacCQ** [comment]
Thank you for pointing this out.

 Looks like we need to do the same for `Initialize` as well.

**Comment by cqc-alec** [comment]
Closed by #202 .
